src/client/types.ts

```diff
@@ -19,3 +19,105 @@ export type HeadersInit =
   | Record<string, string>
   | [string, string][]
   | IterableIterator<[string, string]>;
+
+export type RequestOptions = RequestInit & { backend?: string };
+
+/**
+ * Chat Types
+ */
+
+export type ChatCompletionMessage = {
```

Comment:
Maybe we should separate chat types to somewhere else, wdyt? Maybe a separate folder for LLM stuff like `/llm/types.ts`